site/en/blog/private-network-access-update/index.md

```diff
@@ -148,15 +155,15 @@ still mention the earlier milestone.
 
 This deprecation is accompanied by a deprecation trial, allowing web developers
 whose websites make use of the deprecated feature to continue using it until
-Chrome 113 by registering for tokens. See [below](#register-deprecation-trial)
+Chrome 117 by registering for tokens. See [below](#register-deprecation-trial)
```

Comment:
This reads to me like they can still use the deprecation feature in Chrome 117.
I looked up the definition of "until" https://ell.stackexchange.com/questions/33340/is-until-inclusive-or-exclusive.  It seems that it's usually inclusive when used positively and exclusive when used negatively.  This is probably why "deprecation will not occur until Chrome 117." sounds like 117 is not included but "continue using it until Chrome 117" sounds like 117 is included.  It is also consistent with how we used it before since in Chrome 113 the deprecation trial was still enabled.
I'm not a native English speaker or a professional UX writer though.

Reply:
Ah, interesting! Thanks for pointing that out, TIL. I'll amend to consistently say 116 then, I think that will be less confusing. Note that the previous "timeline" section mistakenly said that the trial would end with the release of 116, though.

Reply:
You're right!  So it was consistent but wrong in a more obvious way.  I'm not entirely sure what I said was right though, so feel free to consult someone else 😆